prompt.py

```diff
@@ -222,11 +222,13 @@ def do_create(self, arguments):
                     print("Exception creating wallet: %s " % e)
                     return
 
-                contract = self.Wallet.GetDefaultContract()
-                key = self.Wallet.GetKey(contract.PublicKeyHash)
-
                 print("Wallet %s " % json.dumps(self.Wallet.ToJson(), indent=4))
-                print("pubkey %s " % key.PublicKey.encode_point(True))
+
+                contract = self.Wallet.GetDefaultContract()
+                if contract is not None:
```

Comment:
Can this ever be None?

Reply:
Apparently so: https://github.com/CityOfZion/neo-python/blob/master/neo/Wallets/Wallet.py#L892

Reply:
If it can be None then we should raise an Exception / show an error here instead! 

The suggested code would make it seem that everything worked, when in fact it didn't. Before, it would throw an exception because it would access properties of None.

Reply:
I agree, before an error would be thrown in `self.Wallet.GetKey` when supplying `None` as argument. For its purpose I believe the solution is valid until there's a proper update on how to notify `GetDefaultContract()` failure.

Reply:
My preference would be to leave the old code here. It would be clear to the to the user that things broke, with this change it wouldn't. Perhaps we should just have `GetDefaultContract` raise an exception instead of returning `None`

Reply:
I think it should raise an exception if no default contract is found